boards/cuav/nora/nuttx-config/scripts/script.ld

```diff
@@ -186,7 +186,10 @@ SECTIONS
 		*(.gnu.linkonce.d.*)
 		CONSTRUCTORS
 		_edata = ABSOLUTE(.);
-	} > AXI_SRAM AT > FLASH
+	    . = ALIGN(8);
```

Comment:
I'd add a comment here as to why this is required

Reply:
@bkueng - Thank you! I mad me realize it is 256 bit - 8 32 bit words